recipes/py_gd/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.1.6" %}
+{% set libgd_version = "2.2.2" %}
+
+package:
+    name: py_gd
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/NOAA-ORR-ERD/py_gd.git
+    git_tag: v{{ version }}
+
+build:
+    skip: True  # [py3k]
```

Comment:
I think there is some confusion here. Adding the skip doesn't preclude one from adding this functionality later. It is merely acknowledging that we can't get it to work at present. The CIs will still be setup. They will just be temporarily disabled and it is easy to reverse.

For instance, I was unable to get `psutil` to build on Windows in `staged-recipes` and so added a [skip]( https://github.com/conda-forge/staged-recipes/pull/389/files#diff-0a65c334b4534a0a0b90087e898055e4R14 ) for that platform. Later in the feedstock, I changed that skip and re-rendered to get Windows platform support in this PR ( https://github.com/conda-forge/psutil-feedstock/pull/4 ). In fact, it took another PR ( https://github.com/conda-forge/psutil-feedstock/pull/2 ) to finish the job. Suffice it to say that feedstock now supports Windows.

Once `libgd` is fixed and you have tested out a PR of your own on the `py_gd` feedstock to add Windows support, you will be able to re-enable it easily. I'm just suggesting we don't waste resources now on failing builds. Does that make sense?